### DISCUSSION TITLE
react-speech-recognition is not taking speech in opera

### DISCUSSION THREAD

**Discussion by ajaysingh7623** [JamesBrill/react-speech-recognition#230] [Q&A] [answered]
![opera](https://github.com/user-attachments/assets/e05ba7a7-023f-4637-b0df-399a8c47bd39)

plz help me for this  @JamesBrill transcript is coming empty

**Comment by mfkrause** [comment, marked as answer]
Opera does not have a working SpeechRecognition API. You'll need to use a third-party API like [Google Cloud Speech-to-Text API](https://cloud.google.com/speech-to-text) and a polyfill.

See https://caniuse.com/speech-recognition
